shared/js/background/background.es6.js

```diff
@@ -18,21 +18,17 @@
 // otherwise FF might miss the onInstalled event
 const events = require('./$BROWSER-events.es6')
 const settings = require('./settings.es6')
-const tabManager = require('./tab-manager.es6')
 
-function Background () {
+settings.ready().then(() => {
     // clearing last search on browser startup
     settings.updateSetting('last_search', '')
 
     var os = 'o'
-    if (window.navigator.userAgent.indexOf('Windows') != -1) os = 'w'
-    if (window.navigator.userAgent.indexOf('Mac') != -1) os = 'm'
-    if (window.navigator.userAgent.indexOf('Linux') != -1) os = 'l'
+    if (window.navigator.userAgent.indexOf('Windows') !== -1) os = 'w'
+    if (window.navigator.userAgent.indexOf('Mac') !== -1) os = 'm'
+    if (window.navigator.userAgent.indexOf('Linux') !== -1) os = 'l'
 
     localStorage['os'] = os
 
     events.onStartup()
-}
-
-var background
-settings.ready().then(() => new Background())
```

Comment:
Lint was complaining that `background` wasn't used. I didn't really see the point in having `background` as a class if all it did was call some things when settings are ready.